components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/login.jsp

```diff
@@ -337,6 +338,21 @@
                                                        "sign.in.with")%> IWA"/>
                                 </a>
                                 <label for="icon-<%=iconId%>">IWA</label>
+                                </div>
+                                <%
+                                    }
+                                    if (localAuthenticatorNames.contains(X509_CERTIFICATE_AUTHENTICATOR)) {
+                                %>
+                                <div>
+                                    <a onclick="javascript: handleNoDomain('<%=Encode.forJavaScriptAttribute(Encode.
+                                forUriComponent(idpEntry.getKey()))%>',
+                                            'x509CertificateAuthenticator')" class="main-link" style="cursor:pointer" id="icon-<%=iconId%>">
+                                        <img class="idp-image" src="images/login-icon.png" data-toggle="tooltip"
+                                             data-placement="top" title="<%=AuthenticationEndpointUtil.i18n(resourceBundle,
+                                                       "sign.in.with")%> x509CertificateAuthenticator"/>
+                                    </a>
+                                    <label for="icon-<%=iconId%>">x509CertificateAuthenticator</label>
```

Comment:
I think better if display text is 'X509 Certificate'

Reply:
done